BuildLoader.py

```diff
@@ -978,6 +981,12 @@ def pre_build(self):
 			setattr(self._board, '%s_SIZE' % each, get_fsp_size(fsp_bin) if self._board.HAVE_FSP_BIN else 0)
 			setattr(self._board, '%s_UPD_SIZE' % each, get_fsp_upd_size(fsp_bin) if self._board.HAVE_FSP_BIN else 0)
 
+		if self._board.BUILD_CSME_UPDATE_DRIVER:
+			if not os.path.exists(os.path.join(sbl_dir, 'Silicon', self._board.SILICON_PKG_NAME, "Library/MeFwUpdateLib/MeFwUpdateLib.lib")):
+				raise Exception  ('BUILD_CSME_UPDATE_DRIVER is enabled, expecting MeFwUpdateLib.lib !')
+			else:
+				if os.name != 'nt':
+					raise Exception  ('BUILD_CSME_UPDATE_DRIVER is enabled, build only works in WINDOWS !')
```

Comment:
I don't think these lines are necessary.
- If file name or directory is changed, BuildLoader.py needs to be changed also.
- If MeFwUpdateLib does not exists, build will eventually fail regardless of this Exception.
- Linux build also will eventually fail. Instead, comment/description would be okay at BUILD_CSME_UPDATE_DRIVER definition(line#196)

Reply:
Linux check is required to inform user about the lib, removed the check for library